[Bug]: Skip Chromium download removed from install.js

### Bug description

https://github.com/puppeteer/puppeteer/blob/v19.0.0/packages/puppeteer/install.js
https://github.com/puppeteer/puppeteer/blob/v19.1.0/packages/puppeteer/install.js

With version 19.1.0 the Code Snippets for checking the skip download variables has been removed completely. 
Instead you moved this code to the configuration and second install file here:

https://github.com/puppeteer/puppeteer/blob/v19.1.1/packages/puppeteer/src/node/install.ts

The skip download is logged and then. Just continued. The return is missing here and you cannot prevent the installation of the browser...

### Puppeteer version

19.1.1

### Node.js version

18.x

### npm version

8.x

### What operating system are you seeing the problem on?

macOS

### Relevant log output

```shell
> [npm-dependencies 1/1] RUN export PUPPETEER_SKIP_DOWNLOAD=true; npm install --only=production nestjs-redoc @nestjs/axios pug puppeteer-cluster bull sharp moment:
#0 23.47 npm ERR! code 1
#0 23.47 npm ERR! path /usr/src/nest-app/node_modules/puppeteer
#0 23.47 npm ERR! command failed
#0 23.47 npm ERR! command sh -c -- node install.js
#0 23.47 npm ERR! **INFO** Skipping browser download as instructed.
#0 23.47 npm ERR! The chromium binary is not available for arm64.
#0 23.47 npm ERR! If you are on Ubuntu, you can install with: 
#0 23.47 npm ERR! 
#0 23.47 npm ERR!  sudo apt install chromium
#0 23.47 npm ERR! 
#0 23.47 npm ERR! 
#0 23.47 npm ERR!  sudo apt install chromium-browser
#0 23.47 npm ERR! 
#0 23.47 npm ERR! ERROR: Failed to set up Chromium r1045629! Set "PUPPETEER_SKIP_DOWNLOAD" env variable to skip download.
#0 23.47 npm ERR! Error
#0 23.47 npm ERR!     at handleArm64 (/usr/src/nest-app/node_modules/puppeteer-core/lib/cjs/puppeteer/node/BrowserFetcher.js:137:11)
#0 23.47 npm ERR!     at BrowserFetcher.download (/usr/src/nest-app/node_modules/puppeteer-core/lib/cjs/puppeteer/node/BrowserFetcher.js:284:13)
#0 23.47 npm ERR!     at async downloadBrowser (/usr/src/nest-app/node_modules/puppeteer/lib/cjs/puppeteer/node/install.js:65:5)
#0 23.47 
#0 23.47 npm ERR! A complete log of this run can be found in:
#0 23.47 npm ERR!     /root/.npm/_logs/2022-10-24T15_08_53_000Z-debug-0.log
```

--- a/packages/puppeteer/src/node/install.ts
+++ b/packages/puppeteer/src/node/install.ts
@@ -39,6 +39,7 @@ export async function downloadBrowser(): Promise<void> {
   const configuration = getConfiguration();
   if (configuration.skipDownload) {
     logPolitely('**INFO** Skipping browser download as instructed.');
+    return;
   }
 
   const product = configuration.defaultProduct!;

